### DISCUSSION TITLE
Help identify issues with MySQL 8.0

### DISCUSSION THREAD

**Discussion by bpamiri** [wheels-dev/wheels#1153] [Q&A] [answered]
Looking to get the framework working on MySQL 8.0. If you have are are using MySQL 8.0 I'd love to hear what you ran into. Right now I've got the test suite passing on Lucee with MySQL 8.0 but ACF 2016 and ACF 2018 are failing.

**Comment by zainforbjs** [comment]
@bpamiri I tried using MySQL 8 with cfwheels in a simple CRUD operation app for users.
The issue is occurring when I go to index of users. This is working correctly for Lucee but gives error when I tried with CF 2016.
CF 2016 is adding some columns that are not present in the table.

This is the issue occurring.
![image](https://user-images.githubusercontent.com/115699497/221481794-2b2b8ae7-5324-466c-8d2f-6c7f2f3ddbdd.png)
![image](https://user-images.githubusercontent.com/115699497/221481441-082a56eb-204b-4254-b386-5b0252062aef.png)

The model for Users which is causing findAll() function.
![image](https://user-images.githubusercontent.com/115699497/221481315-40fbfecb-5d5f-4175-bc41-f28a1761d32a.png)

System Info:
![image](https://user-images.githubusercontent.com/115699497/221482293-c1ab25d0-bcdd-4557-ba4a-a7b4ac218f90.png)

**Comment by zainforbjs** [comment]
@bpamiri I also encountered another issue while using MySQL 8 with Lucee. It renders the view correctly for users but when I try to dump the users' query, it shows a blank screen and does not output the dump.
I have also enabled the Lucee debugging the admin.

This is what I am trying to dump.
![image](https://user-images.githubusercontent.com/115699497/221539394-59dcf8cb-32d0-4d8b-9d13-c7f81af4d2f0.png)

If I remove the dump, the index is correctly rendered.

Don't know if there is some setting needed to be enabled or it is a MySQL 8 issue. Will try with downgrading MySQL and try again

**Comment by neokoenig** [comment]
Not sure how progress is on this but MySQL 5.7 is now EOL, and AWS are dropping RDS support in February.

**Comment by bpamiri** [comment, marked as answer]
@neokoenig we've actually been testing against MySQL latest for a while now. I just finished some work on the test suites and got Adobe 2021 and Adobe 2023 integrated. The current test suite is definitely using MySQL latest so its some flavor of 8.x. You can check out the latest dev built using:
`box install cfwheels-base-template@2.5.0-dev`
